src/Serializer/SerializerContextBuilder.php

```diff
@@ -52,12 +52,19 @@ public function createFromRequest(Request $request, bool $normalization, array $
         } elseif (isset($attributes['item_operation_name'])) {
             $operationKey = 'item_operation_name';
             $operationType = OperationType::ITEM;
+        } elseif (isset($attributes['resource_operation_name'])) {
+            $operationKey = 'resource_operation_name';
+            $operationType = 'resource';
```

Comment:
Then we should introduce a constant in `OperationType` as for other types.

Reply:
It's not really an operation, that's why I don't think it's a good idea to add it to `OperationType`.
Moreover, I can't add it to the `TYPES`, since there is no service corresponding to it.

Reply:
I've added it anyway but I find it strange.